aztec/src/test/kotlin/org/wordpress/aztec/MarkSpanTest.kt

```diff
@@ -0,0 +1,52 @@
+package org.wordpress.aztec
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.wordpress.aztec.spans.MarkSpan
+
+@RunWith(RobolectricTestRunner::class)
+class MarkSpanTest {
+    /**
+     * Test used to confirm two crashes related are fixed.
+     *
+     * https://github.com/wordpress-mobile/WordPress-Android/issues/20738
+     */
+    @Test
+    fun `Calling MarkSpan#safelyParseColor with empty string should not cause a crash`() {
+        var error = false
+        var result: Int? = null
+        try {
+            val span = mockk<MarkSpan>()
+            every { span.safelyParseColor("") } coAnswers { callOriginal() }
```

Comment:
Using `answers` here should be enough since the method is not a suspend function.